smithy-kotlin-codegen/src/main/kotlin/software/amazon/smithy/kotlin/codegen/core/RuntimeTypes.kt

```diff
@@ -206,8 +206,8 @@ object RuntimeTypes {
                 val SigningEndpointProvider = runtimeSymbol("SigningEndpointProvider", KotlinDependency.AWS_SIGNING_COMMON)
             }
 
-            object AwsSigningCrt {
```

Comment:
Shouldn't we remove the CRT implementation entirely? ie. `runtime/auth/aws-signing-crt`

Reply:
No we intend to offer the CRT as an alternate signing implementation for the foreseeable future. Currently, for example, the new implementation doesn't support SigV4a. There may be other situations in which people want (or need) to use the CRT here.